tests/integration/tester-grpc/src/resources/webserver.py

```diff
@@ -168,7 +167,7 @@ def init_overlord():
                     print(f"received {resp} from {subordinate}")
                     try:
                         id_, value_ = resp.text.split("=")
-                    except:
+                    except:  # noqa
```

Comment:
Is this change merely because `except:  # noqa` is shorter than `except Exception:`?

Or are you expecting some weird exception like BaseException or forbidden non-exception-ey base?

Reply:
no I think that was pure laziness. It's testing code so we didn't mind too much about linting.